src/partr.c

```diff
@@ -125,7 +125,11 @@ void jl_parallel_gc_threadfun(void *arg)
 
     // initialize this thread (set tid and create heap)
     jl_ptls_t ptls = jl_init_threadtls(targ->tid);
-
+    void *stack_lo, *stack_hi;
+    jl_init_stack_limits(0, &stack_lo, &stack_hi);
+    // warning: this changes `jl_current_task`, so be careful not to call that from this function
+    jl_task_t *ct = jl_init_root_task(ptls, stack_lo, stack_hi);
+    JL_GC_PROMISE_ROOTED(ct);
```

Comment:
Is this needed? `jl_init_root_task` does that [here](https://github.com/RelationalAI/julia/blob/v1.10.2%2BRAI/src/task.c#L1685).

Reply:
Better to keep it as it is, given it's the idiom used upstream and in the other thread functions.